_data/news.yml

```diff
@@ -0,0 +1,4 @@
+- name: Open Science FAIR
+  date: 2021-09-20
+  description: Test
```

Comment:
@korbinib @lauportell @floradanna @smza @wittigue 

Could someone help me out with Adding some news items by commenting or changing this yaml file? Thanks!

Reply:
multi day events don't have an end-date, do they?

Reply:
Well I wanted to not let it look like an event ... Because it is a news item :)

Reply:
The date should actually be the date the news item is added

Reply:
So all the info you want to include has to be done through the description attribute.

Reply:
yes, will add the ones I know